tensorboard/webapp/notification_center/_views/view_types.ts

```diff
@@ -17,7 +17,12 @@ import {Notification} from '../_redux/notification_center_types';
 /**
  * A notification chunk displayed in the notification center.
  */
-
 export interface ViewNotificationExt extends Notification {
   icon: string | null;
+  hasRead: boolean;
 }
+
+/**
+ * The key used to record the latest read timestamp in local storage.
+ */
+export const NOTIFICATION_LAST_READ_DATE_KEY = 'notificaitonLastReadDateInMs';
```

Comment:
prob should be defined in the store type OR data_source type. Definitely should not be defined in the view types.

Reply:
This was accidentally left when I move the key to _redux. It is not used anywhere now. Removed.